README.md

```diff
@@ -1 +1,259 @@
-# Hello-Microverse-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Set up a "Hello Microverse" Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Set up a "Hello Microverse" project <a name="about-project"></a>
+
+
+**Set up a "Hello Microverse" project** is my first project on Microverse Program, It is focused on learning some important issues that will be used along all program experience.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>GitHub flow</summary>
+  <ul>
+    <li>GitHub flow</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Commit messages</summary>
+  <ul>
+    <li>Commitmessages</li>
+  </ul>
+</details>
+
+<details>
+<summary>Pull Requests</summary>
+  <ul>
+    <li>Pull Requests</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Creating branches on repositories.**
+- **Use of linter on html&css project.**
+- **Write a descriptive README file.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
```

Comment:
Heyy 👋🏽 this is good to have but only if you have a deployed application. If the link isn't working I'd suggest removing it and mentioning that or just adding `N/A`. 👍🏽